pkg/util/env/env.go

```diff
@@ -33,6 +33,8 @@ const (
 	antreaCloudEKSEnvKey = "ANTREA_CLOUD_EKS"
 
 	defaultAntreaNamespace = "kube-system"
+
+	allowNoEncapWithoutAntreaProxy = "ALLOW_NO_ENCAP_WITHOUT_ANTREA_PROXY"
```

Comment:
to match other constants above, this should be `allowNoEncapWithoutAntreaProxyEnvKey` in theory. But IMO, defining a constant like this for a string which is only used once is not really needed.

Reply:
To avoid creating this constant, we need to directly use `getBoolEnvVar` to get `ALLOW_NO_ENCAP_WITHOUT_ANTREA_PROXY`, but this way may cause format confusion.
So IMO, it is better to change it to `allowNoEncapWithoutAntreaProxyEnvKey `.